clients/banking/src/components/TransactionDetail.tsx

```diff
@@ -139,9 +139,10 @@ export const TransactionDetail = ({ transaction, large }: Props) => {
           style={styles.tile}
           footer={match(transaction)
             .with(
-              { type: "SepaCreditTransferOut" },
-              { type: "SepaCreditTransferIn" },
-              { originTransactionId: P.string },
+              {
+                type: "SepaCreditTransferOut" || "SepaCreditTransferIn",
```

Comment:
unfortunatelly this will not work if type is `"SepaCreditTransferIn"`  
Because the code you write will always generate the following object:
```ts
{
  type: "SepaCreditTransferOut",
  originTransactionId: P.string,
}
```
because `"SepaCreditTransferOut" || "SepaCreditTransferIn"` always return `"SepaCreditTransferOut"` (you can test it in your terminal by running `node`).

The solution is using `P.union("SepaCreditTransferOut", "SepaCreditTransferIn")`